localization/simp_chinese/0_countries_l_simp_chinese.yml

```diff
@@ -238,7 +238,7 @@
 #  FLA_ADJ:0 "Flemish"
 #  FNG:0 "Fang"
 #  FNG_ADJ:0 "Fang"
- FRA:0 "法国"
+ FRA:0 "法兰西共和国"
```

Comment:
You might want to consider using the generic name instead as the FRA loc would also apply to France if it is a monarchy.

Reply:
Fixed in next PR.